index.html

```diff
@@ -122,6 +122,33 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Paul Kim</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
```

Comment:
Please note that you need to update your profile link ( github.com/paul ) instead of the platform link ( eg., github.com ) in these 'contact' section. Update those links and make a commit.